src/exchanges/upbit-client.js

```diff
@@ -0,0 +1,320 @@
+const semaphore = require("semaphore");
+const BasicClient = require("../basic-client");
+const BasicMultiClient = require("../basic-multiclient");
+const Ticker = require("../ticker");
+const Trade = require("../trade");
+const Level2Point = require("../level2-point");
+const Level2Snapshot = require("../level2-snapshot");
+const Level2Update = require("../level2-update");
+
+class UpbitClient extends BasicMultiClient {
```

Comment:
Do we need to use BasicMultiClient?

Reply:
yes, it supports several pairs for a single socket, but in some other way than the others - it overwrites previous values. Here is an example:

Call this:

```
this._wss.send(
        JSON.stringify([{"ticket":"concluded"},{"type":"trade","codes":["BTC_USD", "LTC_USD", "BTT_USD"]}])
      );
```

Returns 3 pairs;

Then adding this:
```
this._wss.send(
        JSON.stringify([{"ticket":"concluded"},{"type":"trade","codes":["XRP_USD"]}])
      );
```
Returns only XRP, not 4 of them.

And for deletion, it's also require to set what "needs to be left", not what "needs to be deleted".

So we should create a new client for this. Please confirm.

Reply:
So it requires sending the active list of markets for both subscribe and unsubscribe events?  
Can't you just iterate on the active markets in the the subscribed market map?  

If you refer to https://github.com/altangent/ccxws/blob/master/src/basic-client.js#L169, the market get added to a map prior to calling the sub-classes subscribe method.  You should be able to construct the complete set of subscribed markets from this.

Reply:
Yes, DONE, client is now basic single